src/FluentSchema.integration.test.js

```diff
@@ -234,4 +236,53 @@ describe('FluentSchema', () => {
       })
     })
   })
+
+  describe('basic.json', () => {
+    it.only('generate', () => {
+      const [step] = basic
+      expect(
+        deepOmit(
+          FluentSchema()
+            .asArray()
+            .title('Product set')
+            .items(
+              FluentSchema()
+                .title('Product')
+                .prop('uuid') // TODO LS bug if we use `id` the property is removed by deepOmit
```

Comment:
@mcollina this is a super weird bug. 
I wrote a quick deepOmit to drop all the **$id** after the schema has been generated so that we can compare the basic.json to our result. Everything works fine apart from the prop('id) which is removed by the deepOmit 🤔. Once renamed from id => uuid everything is good.

Reply:
Regardless, IMO isn't a big deal because deepOmit is used only for testing purpose